game_event/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from . import views
+
+
+urlpatterns = [
+    path('game-event/', views.game_events, name='game-events')
+    ]
```

Comment:
Do you still need this? 

i dont know which PR will be merged first, so in a case this one get's merged, i will have to do quick rebase to include the game_event.urls in the meat_balls_urls

so i wonder maybe you can just fix it now  ? you can look at my PR #69 

and instead of the 'game-event' routing you got there you could replace that line with 
`path('', views.game_events, name='game events')`

the empty string in the path is because it has the prefix of "game-events/" from the meat_balls.urls

Reply:
I'll change today according to your PR. I forgot to do it after we talked about it.

Reply:
@Danielsio can this one be resolved?